api/rooms/views/map.py

```diff
@@ -0,0 +1,116 @@
+import json
+from pyproj            import Transformer
+
+from django.db.models  import Q
+from django.views      import View
+from django.http       import JsonResponse
+
+from ..models          import Room, TradeType
+
+def wgs84_to_utmk(longitude, latitude):
+    return Transformer.from_crs("epsg:4326", "epsg:5178").transform(latitude, longitude)
+
+def utmk_to_wgs84(utmk_x, utmk_y):
+    return Transformer.from_crs("epsg:5178", "epsg:4326").transform(utmk_x, utmk_y)
+
+def make_room_filter(func):
+    def wrapper(self, request, *args, **kwargs):
+        filter = json.loads(request.GET.get('filter', '{}'))
+        room_type  = filter.get('roomType', [])
+        trade_type = filter.get('tradeType', [])
+        locations  = json.loads(request.GET.get('location', '[]'))
+        dong_code  = request.GET.get('dongCode')
```

Comment:
변수명을 복수로만 수정해주시면 될 것 같아요!

```suggestion
        filters     = json.loads(request.GET.get('filter', '{}'))
        locations   = json.loads(request.GET.get('location', '[]'))
        dong_code   = request.GET.get('dongCode')
                
        room_types  = filters.get('roomType', [])
        trade_types = filters.get('tradeType', [])
```

Reply:
수정완료하였습니다.